src/plugin/fdpp/fdpp.c

```diff
@@ -86,6 +87,14 @@ static void fdpp_call_noret(struct vm86_regs *regs, uint16_t seg,
     *regs = REGS;
 }
 
+static void fdpp_relocate_notify(uint16_t oldseg, uint16_t newseg,
+                                 uint16_t startseg, uint32_t blklen)
```

Comment:
startseg??

Reply:
Oh boy, that's not right. Too much of a hurry this morning, standby.